tensorboard/webapp/deeplink/hash.ts

```diff
@@ -23,25 +23,28 @@ const TAB = '__tab__';
 
 @Injectable()
 export class HashDeepLinker implements DeepLinkerInterface {
-  private readonly tfStorage: TfStorageElement;
+  tfStorage: DeepLinkerInterface & {
```

Comment:
Is this an accurate representation of tfStorage's type? It doesn't have getPluginId() or setPluginId() as far as I can tell. tb_polymer_interop_types already defines a TfStorage type, which might be more appropriate?

(While you're at it, should we also remove the TfStorageElement and TfGlobalElement types from tb_polymer_interop_types?)

Reply:
This has been changed so significantly that I no longer believe this is a relevant comment.